CONTRIBUTING.md

```diff
@@ -1,16 +1,28 @@
+# Contributing to Riot.js
 
-### Coding style
+If you’d like to contribute a feature or bug fix, you can [fork](https://help.github.com/articles/fork-a-repo/) Riot, commit your changes, & [send a pull request](https://help.github.com/articles/using-pull-requests/).
+Please make sure to [search the issue tracker](https://github.com/mutt/riotjs/issues) first.
 
-The most important:
+### Development Process
 
-- Leave out semicolons
-- Use single quotes and not double quotes
-- Prefer `==` over `===` unless it's a must
-- Spacing around brackets: `if (foo) {` instead of `if(foo){`
+`master` is considered safe and should be identical to the last release tag. When submitting pull requests please use the `dev` branch.
 
+### Coding Guidelines
 
-### Use `dev` branch
+Please follow the coding style established in the rest of the codebase.
 
-Please start from `dev` branch and hand your work with a **pull request**.
+  **Spacing**:<br>
+  Use two spaces for indentation. No tabs.
+  Spacing around brackets: `if (foo) {` instead of `if(foo){`
 
-##### Thank you in advance! We are happy to merge your code.
\ No newline at end of file
+  **Quotes**:<br>
+  Single-quoted strings are preferred to double-quoted strings.
+
+  **Equality Checking**:<br>
+  Prefer `==` over `===` unless it's a must.
```

Comment:
I was pretty surprised when I saw this since it goes pretty hard against convention. Did you mean for this to be the other way around? I agree that they are both valid tools and you shouldn't prevent type coercion with `==` but to prefer it was very surprising.